openghg/analyse/_scenario.py

```diff
@@ -88,6 +88,9 @@ class ModelScenario:
     operations to be performed combining these inputs.
     """
 
+    def __bool__(self) -> bool:
```

Comment:
Does this only return True if all of `obs`, `footprint`, `fluxes` and `bc` are True? For this objects it's actually ok if some of these are missing depending on what operations are being performed so this could be an `or` so at least it's true if *something* is attached?

Reply:
OK done